File: kuba/src/app/patient.ts
export class Patient {
    Firstname: string;
    Lastname: string;
    Age: number;
    Address: string;
    Description: string;
    /*tel: string;
  summary_quest: number;
  summary_test: number;
  */
}

export interface IPatient {
    Firstname: string;
    Lastname: string;
    Age: number;
    Address: string;
    Description: string;
}

File: kuba-api/kuba-api/Models/Patient.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace kubaapi.Models
{
    public class Patient
    {
        public long Id { get; set; }
        public string Firstname { get; set; }
        public string Lastname { get; set; }
        public int Age { get; set; }
        public string Description { get; set; }
        public string Address { get; set; }
    }
}
